lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -188,6 +187,20 @@ class DetailsRenderer {
     return element;
   }
 
+  /**
+   * @param {string} type
+   * @param {*} value
+   */
+  _renderUnknown(type, value) {
+    // eslint-disable-next-line no-console
+    console.error(`Unknown valueType: ${type}`, value);
+    const element = this._dom.createElement('details', 'lh-unknown');
+    this._dom.createChildOf(element, 'summary').textContent =
+      `Details type '${type}' unrecognized by this version of the report renderer.`;
```

Comment:
`Ruh roh! 😿 We don't know how to render audit details of type \`${type}\`. The Lighthouse version that collected this data is likely newer than the Lighthouse version of the report renderer. Expand for the raw JSON.`

Reply:
Can we not include the `Ruh roh!` please :)

Reply:
Having long text here will be bad if the unknown type is in a table. Thoughts on just "Unknown" -> expand -> see the message you have here followed by the JSON?

Reply:
> Having long text here will be bad if the unknown type is in a table. Thoughts on just "Unknown" -> expand -> see the message you have here followed by the JSON?

how does the pre-box look in that case (especially if there's one on each row or whatever)? Inside a table may be a lost cause regardless

Reply:
> Inside a table may be a lost cause regardless

yea, I guess this is fine then.